### PR TITLE
Extract whether the comment is pinned

### PR DESCRIPTION
**Pull request by B0pol**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [ ] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

### CODE REVIEW

**Review by TobiGr — APPROVED** [review]

File: extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItem.java [hunk 1]
```diff
@@ -94,4 +95,12 @@ public void setHeartedByUploader(boolean isHeartedByUploader) {
     public boolean getHeartedByUploader() {
         return this.heartedByUploader;
     }
+
+    public boolean getPinned() {
```
Comment (TobiGr) [inline extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItem.java:99]: Ah damn. I was too fast. That should be `isPinned` (and `isHeartedByUploader`)

Comment (opusforlife2) [reply extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItem.java:99]: You could just click on `Revert`?

Comment (B0pol) [reply extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItem.java:99]: No, I'll create a new PR for both

Comment (B0pol) [reply extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItem.java:99]: https://github.com/TeamNewPipe/NewPipeExtractor/pull/525

**Pull request merged**